scripts/run-build-visit

```diff
@@ -273,9 +276,9 @@ elif [ "$lassen" = true ]; then
        PAR_COMPILER_CXX=/usr/tce/packages/spectrum-mpi/spectrum-mpi-rolling-release-gcc-8.3.1/bin/mpicxx \
        PAR_INCLUDE=-I/usr/tce/packages/spectrum-mpi/ibm/spectrum-mpi-rolling-release/include \
        ./${build_visit_script} --group ${dest_group} --required --optional --mesagl \
-       --no-gdal --no-openexr --no-embree --no-ispc --no-tbb --no-pidx \
-       --no-ospray --parallel --qt --qwt --no-pyside --no-visit --no-boost --makeflags -j16 \
-       --thirdparty-path ${dest_dir}
+       --no-anari --no-boost --no-gdal --no-openexr --no-ospray --no-pidx --no-pyside \
```

Comment:
Doesn't --mesagl need --llvm?

Reply:
`llvm` is part of `optional`, so it doesn't need to be specified in this case.